### PR TITLE
Contributing Code page changes

### PR DESCRIPTION
**Pull request by mDuo13**
- Drop the raw list of repos (lists tend to be high maintenance and potentially political)
- Add note on DevNullProd's source code guide
- Add description for community standards drafts

### CODE REVIEW

**Comment by miguelportilla** [comment]
Nice cleanup! One thing to be aware of is that the Source Code Guide is based on version 1.5.0 and will diverge with newer versions. 👍

File: content/contributing/contribute-code.md [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-[ripple-keypairs]: https://github.com/ripple/ripple-keypairs
-[rippled-historical-database]: https://github.com/ripple/rippled-historical-database
-[standards-drafts]: https://github.com/xrp-community/standards-drafts
-[Xpring4j]: https://github.com/xpring-eng/xpring4j
-[XpringKit]: https://github.com/xpring-eng/xpringkit
-[Xpring-JS]: https://github.com/xpring-eng/xpring-js
-[XRP-API]: https://github.com/xpring-eng/xrp-api
-[xrpl-dev-portal]: https://github.com/ripple/xrpl-dev-portal
-[XUMM-App]: https://github.com/XRPL-Labs/XUMM-App
+The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it. Look for "[Source]" links in the [documentation](docs.html) to jump directly into the related source code, or browse the source code on GitHub:
```
Comment (seelabs) [inline content/contributing/contribute-code.md]: I'd love some phrase that encourages participation. I'm not sure what. Maybe: "...Ledger is open-source. Community participation makes it stronger. Anyone can download..."

**Review by seelabs — APPROVED** [review]
Love it. Left one suggestion, but :+1: either way.

**Comment by mDuo13** [comment]
> One thing to be aware of is that the Source Code Guide is based on version 1.5.0 and will diverge with newer versions. +1

Yep, I noticed that. Fortunately the guide itself is tagged with that information. I'm hoping that this brings a little more visibility to DevNullProd's excellent efforts and maybe that'll motivate an effort to keep it up-to-date. But also, my impression is that the main code structure doesn't change rapidly, so even if it's a couple versions out of date it's still a very useful look into the overall way the server works.

**Pull request merged**